addons/atlassian-jira/uninstall.js

```diff
@@ -3,16 +3,22 @@ This is the uninstallation logic of the Lifecycle Manager.
 */
 
 const Sdk = require('@fusebit/add-on-sdk');
-const Superagent = require('superagent');
 
 module.exports = async (ctx) => {
```

Comment:
Consider try/catch around each failable call - few things should cause a deletion to fail in a way that each resource doesn't get at least a best-effort deletion.

Reply:
We decided to fail here so we have visibility at least for now into if/how things are failing. Also, this file is uniform across all addons at the moment. I'm not opposed to switching to something more forgiving there, but not as part of this PR.